src/main/java/com/garlicbread/includify/entity/volunteer/Volunteer.java

```diff
@@ -16,21 +18,36 @@ public class Volunteer {
     private String id;
 
     @Column(nullable = false)
+    @NotBlank(message = "Name is required")
     private String name;
 
     @Column(nullable = false, unique = true)
+    @Email(message = "Please provide email with valid format")
+    @NotBlank(message = "Email is required")
     private String email;
 
     @Column(nullable = false)
+    @NotBlank(message = "Password is required")
     private String hashedPassword;
 
     @Column(nullable = false)
+    @Min(value = 15, message = "Age must be at least 15")
     private int age;
 
     @Column(nullable = false)
+    @NotBlank(message = "Address is required")
     private String address;
 
     @Column(nullable = false)
+    @Pattern(regexp = "^\\+?[0-9. ()-]{7,25}$", message = "Please provide valid phone")
     private String phone;
-}
 
+    public String getHashedPassword() {
```

Comment:
Can you please move these methods to a common Utils class under a new utils package? These will be helpful for Users and Organisation too.